### PR TITLE
rename file and package for client interceptors

### PR DESCRIPTION
**Pull request by NishanthSpShetty**

### CODE REVIEW

File: client/traceidinjector.go [hunk 1]
```diff
@@ -33,5 +33,5 @@ func generateFallBackTraceId() string {
 }
 
 func WithTracingInterceptor(logger log.Logger) grpc.DialOption {
```
Comment (NishanthSpShetty) [inline client/traceidinjector.go]: why do we need to have this here ? can we just expose the interceptor and let the client/user handle what he wants to do with this?
This limits chaining interceptors at the client side. @trivishnu

Comment (sid255) [reply client/traceidinjector.go]: +1

**Comment by sid255** [comment]
Also, can we call the folder clientInterceptors or something similar as client gives the wrong meaning?

**Comment by trivishnu** [comment]
> Also, can we call the folder clientInterceptors or something similar as client gives the wrong meaning?

we already have "interceptor" in library name. still we need to specify interceptor in folder name?

**Comment by NishanthSpShetty** [comment]
> 

the context of the package should be sufficient, would make no sense to keep on adding module name in sub packages.

**Review by trivishnu — APPROVED** [review]

File: client/traceidinjector.go (hunk 1 above)
Comment (NishanthSpShetty) [reply client/traceidinjector.go]: removed as per discussion with @trivishnu .

**Pull request merged**